publication-event-handlers/src/test/java/no/unit/nva/publication/events/doirequests/DoiRequestEventProducerTest.java

```diff
@@ -94,18 +82,6 @@ public void handleRequestThrowsExceptionWhenEventContainsPublicationWithoutIdent
         assertThat(exception.getMessage(), is(equalTo(NO_RESOURCE_IDENTIFIER_ERROR)));
     }
 
-    @Test
-    void handlerCreatesNewDraftDoiEventWhenThereIsNoPreviousDoiRequestAndThereIsNoDoi()
-        throws JsonProcessingException {
-        String eventInput = IoUtils.stringFromResources(Path.of(RESOURCE_UPDATE_NEW_IMAGE_ONLY_WITHOUT_DOI));
-        assertFalse(hasDoiField(eventInput));
-
-        handler.handleRequest(IoUtils.stringToStream(eventInput), outputStream, context);
-        PublicationHolder actual = outputToPublicationHolder(outputStream);
-        assertThat(actual.getType(), is(equalTo(DoiRequestEventProducer.TYPE_REQUEST_FOR_NEW_DRAFT_DOI)));
-        assertThat(actual.getItem(), notNullValue());
-    }
-
     @Test
     void handlerCreatesUpdateDoiEventWhenThereIsNoPreviousDoiRequestButThereIsDoi() throws JsonProcessingException {
```

Comment:
I am writing here although this refers to the file "resource_update_event_new_image_only_with_doi.json".
The event in the file is of type DoiRequest. But an imported Resource with a doi does not have an associated DoiRequest entry. Here the event is a "resource update". that should be forwarded to the DoiRequestEventProducer. The DoiRequestEventProducer will accept the DynamoEntryUpdateEvent and will transform the contents to a Publication (ResourceUpdate has such method for this case exactly).  

Then there are the following cases:
1. Publication(newData) without DOI and with a DoiRequest with status REQUESTED -> creates DraftDoi
2. Publication(newData) with DOI and with a DoiRequest with status REQUESTED -> does nothing
3. Publication(newData) with DOI and with a DoiRequest with status ACCEPTED -> updates DOI metadata
4. Publication(newData) with DOI but WITHOUT a DoiRequest at all -> updates DOI metadata

The problem with case 4 is that since the Resource does not contain the  DoiRequest but it contains the draft doi, and update of a resource with a Draft DOI will create an event of case 4 and  it is  impossible to know if the DOI is draft or published  and its metadata should be updated or not.  

My suggestion is that we create a bug for the already registered DOIs and we currnetly handle only the dois that were created by our own platform